### PR TITLE
Update The HTML Course

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
- [x] Convert SVG->PNG
- [x] Look through comments and fix any mistakes, grammar and such
- [x] Change link names and link `identifiers` (website/article/documentation...`

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/web/html/html-basics/headings-and-paragraphs.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/web/html/html-basics/headings-and-paragraphs.md?ref=f687519001d02f556c08208b24e7ad80ebae260d)

File: web/html/citation-elements/address.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ aspects:
   - deep
 
 links:
-  - '[MDN docs for address](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address){website}'
+  - '[MDN docs for address](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address){documentation}'
```
Comment (kapnobatai137) [inline web/html/citation-elements/address.md]:
```suggestion
  - '[HTML <address> element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address){documentation}'
```

File: web/html/citation-elements/blockquotes.md [hunk 2]
```diff
@@ -21,7 +21,8 @@ aspects:
 
 
 links:
-  - '[MDN docs for blockquotes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote){website}'
+  - '[MDN docs for blockquotes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote){documentation}'
```
Comment (kapnobatai137) [inline web/html/citation-elements/blockquotes.md]: can we have the resource title be the same/similar to the title found on the website? also, there is no need to say MDN docs, just add the {documentation} tags.

```suggestion
  - '[HTML <blockquote> Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote){documentation}'
```

File: web/html/citation-elements/intro-bdi.md [hunk 3]
```diff
@@ -23,8 +23,8 @@ aspects:
   - obscura
 
 links:
-  - '[MDN docs for bdi](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdi){website}'
-  - '[MDN docs for User Agent](https://developer.mozilla.org/en-US/docs/Glossary/user_agent){website}'
+  - '[MDN docs for bdi](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdi){documentation}'
+  - '[MDN docs for User Agent](https://developer.mozilla.org/en-US/docs/Glossary/user_agent){documentation}'
 
 ---
 # bdi Element
```
Comment (kapnobatai137) [inline web/html/citation-elements/intro-bdi.md]: how does this sound?

```suggestion
# The <bdi> Element
```

File: web/html/citation-elements/intro-bdi.md [hunk 4]
```diff
@@ -23,8 +23,8 @@ aspects:
   - obscura
 
 links:
-  - '[MDN docs for bdi](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdi){website}'
-  - '[MDN docs for User Agent](https://developer.mozilla.org/en-US/docs/Glossary/user_agent){website}'
+  - '[MDN docs for bdi](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdi){documentation}'
+  - '[MDN docs for User Agent](https://developer.mozilla.org/en-US/docs/Glossary/user_agent){documentation}'
```
Comment (kapnobatai137) [inline web/html/citation-elements/intro-bdi.md]: no need to say "MDN docs"
first title could be HTML <bdi> Element
second could be just User Agent

File: web/html/citation-elements/intro-bdo.md [hunk 5]
```diff
@@ -23,7 +23,7 @@ aspects:
   - obscura
 
 links:
-  - '[MDN docs for bdo](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdo){website}'
+  - '[MDN docs for bdo](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdo){documentation}'
```
Comment (kapnobatai137) [inline web/html/citation-elements/intro-bdo.md]:
```suggestion
  - '[HTML <bdo> Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdo){documentation}'
```

File: web/html/citation-elements/intro-bdo.md [hunk 6]
```diff
@@ -32,7 +32,7 @@ links:
 
 The HTML `<bdo>`, or the **Bidirectional Text Override** element, is used to override the current directionality of text. This means that you can have text written in one direction, render and display in the opposite direction.
 
-This element has 2 element related attributes, `ltr` and `rtl`.
+This element has 1 element related attribute, `dir` and `dir` has 2 possible values, `ltr` and `rtl`.
```
Comment (kapnobatai137) [inline web/html/citation-elements/intro-bdo.md]: this phrase is a bit of a mouthful. what do you think of this:

```suggestion
This element can have only one attribute, `dir`, with only 2 two possible values:```

File: web/html/citation-elements/intro-cite.md [hunk 7]
```diff
@@ -20,7 +20,7 @@ aspects:
   - deep
 
 links:
-  - '[MDN docs for cite](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite){website}'
+  - '[MDN docs for cite](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite){documentation}'
```
Comment (kapnobatai137) [inline web/html/citation-elements/intro-cite.md]: no need for "MDN docs" here

File: web/html/citation-elements/intro-q.md [hunk 8]
```diff
@@ -20,7 +20,7 @@ aspects:
   - deep
 
 links:
-  - '[MDN docs for q](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/q){website}'
+  - '[MDN docs for q](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/q){documentation}'
```
Comment (kapnobatai137) [inline web/html/citation-elements/intro-q.md]: no need for "MDN docs" here

File: web/html/citation-elements/intro-q.md [hunk 9]
```diff
@@ -20,7 +20,7 @@ aspects:
...
 
 ---
 # Intro q
```
Comment (kapnobatai137) [inline web/html/citation-elements/intro-q.md]: should we reference elements like <q>?

```suggestion
# Intro <q>
```

File: web/html/formatting-text-elements/important-b-and-strong.md [hunk 10]
```diff
@@ -18,8 +18,8 @@ aspects:
   - introduction
 
 links:
-  - '[MDN docs for b](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b){website}'
-  - '[MDN docs for strong](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong){website}'
+  - '[MDN docs for b](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b){documentation}'
+  - '[MDN docs for strong](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong){documentation}'
 
 ---
 # Important (b & strong)
```
Comment (kapnobatai137) [inline web/html/formatting-text-elements/important-b-and-strong.md]: would this look better?

```suggestion
# Important (<b> & <strong>)
```

File: web/html/forms-ii/radio-buttons.md [hunk 11]
```diff
@@ -17,7 +17,7 @@ aspects:
   - workout
 
 links:
-  - '[MDN docs for radio buttons](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio){website}'
+  - '[MDN docs for radio buttons](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio){documentation}'
```
Comment (kapnobatai137) [inline web/html/forms-ii/radio-buttons.md]: can you do a grep & replace for all the instances of "MDN docs" and remove them? Also tweak the titles a bit. One way you could do it is to make the description be close to the title of the article/documentation/discussion

```suggestion
  - '[MDN docs for radio buttons](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio){documentation}'
```

File: web/html/links/link-mailto.md [hunk 12]
```diff
@@ -108,3 +108,4 @@ Email Me
 * mail:
 * address:
 * email:
+0
```
Comment (kapnobatai137) [inline web/html/links/link-mailto.md]: I think you inserted this 0 by mistake
```suggestion
```

File: web/html/sectioning-elements/footer.md [hunk 13]
```diff
@@ -19,7 +19,7 @@ aspects:
   - workout
 
 links:
-  - '[MDN: Footer](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer){website}'
```
Comment (kapnobatai137) [inline web/html/sectioning-elements/footer.md]: I think I like the resource description better how it was before. We don't need to say "docs" when we use the `documentation` tag.

File: web/html/tables/intro-th.md [hunk 14]
```diff
@@ -28,7 +28,7 @@ links:
   - '[CodePen: Table multiple spans](https://codepen.io/enkidevs/pen/XBdpgW){code}'
   - '[CodePen: Table multiple spans](https://codepen.io/enkidevs/pen/NBNdwO){code}'
   - '[CodePen: Table abbr attribute](https://codepen.io/enkidevs/pen/ajNJdW){code}'
-  - '[MDN docs for th](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th){website}'
+  - '[MDN Docs On th](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th){documentation}'
```
Comment (kapnobatai137) [inline web/html/tables/intro-th.md]: how about this:
```suggestion
  - '[MDN: <th> Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th){documentation}'
```

**Comment by kapnobatai137** [comment]
main takeaway here is that:
- if you have 2 similar resources from different websites (e.g. MDN & W3S) you should name them like "MDN:  Element", "W3S:  Element"
- if there is only 1 website (e.g. MDN), you should just name them "HTML  Element" or " Element"
- there is no need to use "docs" in the description when the tag is `{documentation}`

@Stefan-Stojanovic

File: web/html/forms-iii/range.md [hunk 15]
```diff
@@ -9,20 +9,16 @@ type: normal
...
   - workout
   - deep
 
 links:
-  - '[Range elements on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range){documentation}'
+  - '[HTML <input type="range"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range){documentation}'
 
 ---
-# Range
+# The HTML `<input type="range"`
```
Comment (kapnobatai137) [inline web/html/forms-iii/range.md]:
```suggestion
# HTML `range` type input
```

File: web/html/forms-iii/type-password.md [hunk 16]
```diff
@@ -21,7 +17,7 @@ aspects:
 links:
 
 ---
-# Type=password
+# The HTML `<input type="password"`
```
Comment (kapnobatai137) [inline web/html/forms-iii/type-password.md]:
```suggestion
# HTML `password` type input
```

File: web/html/media/video-formats.md [hunk 17]
```diff
@@ -9,16 +9,13 @@ type: normal
...
-  web.embed-media.1: 10
-
 aspects:
   - introduction
   - workout
   - deep
 
 links:
-  - '[a walkthrough](https://www.quackit.com/html_5/tags/html_video_tag.cfm){website}'
+  - '[Walkthrough For <video>](https://www.quackit.com/html_5/tags/html_video_tag.cfm){article}'
```
Comment (kapnobatai137) [inline web/html/media/video-formats.md]:
```suggestion
  - '[Walkthrough for <video>](https://www.quackit.com/html_5/tags/html_video_tag.cfm){article}'
```

File: web/html/media/video-subtitles.md [hunk 18]
```diff
@@ -9,17 +9,14 @@ type: normal
...
-
 aspects:
   - introduction
   - workout
   - deep
 
 links:
-  - '[a walkthrough](https://www.quackit.com/html_5/tags/html_video_tag.cfm){website}'
-  - '[Documentation on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video){documentation}'
+  - '[Walkthrough For <video>](https://www.quackit.com/html_5/tags/html_video_tag.cfm){article}'
```
Comment (kapnobatai137) [inline web/html/media/video-subtitles.md]:
```suggestion
  - '[Walkthrough for <video>](https://www.quackit.com/html_5/tags/html_video_tag.cfm){article}'
```

File: web/html/web-page-structure/optional-elements.md [hunk 19]
```diff
@@ -9,24 +9,19 @@ type: normal
...
-  web.syntax-html.4: 10
-  web.syntax-html.5: 10
-
 aspects:
   - introduction
 
 links:
-  - '[w3 documents on all Optional Elements](https://www.w3.org/TR/html5/syntax.html#optional-tags){website}'
-  - '[Google docs for Optional Elements](https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags){website}'
+  - '[w3: Optional Elements](https://www.w3.org/TR/html5/syntax.html#optional-tags){documentation}'
```
Comment (kapnobatai137) [inline web/html/web-page-structure/optional-elements.md]:
```suggestion
  - '[W3: Optional Elements](https://www.w3.org/TR/html5/syntax.html#optional-tags){documentation}'
```

File: web/html/web-page-structure/optional-elements.md [hunk 20]
```diff
@@ -9,24 +9,19 @@ type: normal
...
+  - '[google.github.io: Optional Elements](https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags){documentation}'
```
Comment (kapnobatai137) [inline web/html/web-page-structure/optional-elements.md]:
```suggestion
  - '[Google: Optional Elements](https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags){documentation}'
```

**Comment by kapnobatai137** [comment]
- [ ] change the colors for all images

**Pull request merged**